packages/env/lib/config/parse-config.js

```diff
@@ -110,6 +120,23 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 		};
 	}
 
+	// Allow shortcut for .zip sources on WordPress.org.
+	const wpOrgShortSource = sourceString.match( /^wp.org\/(.*)/ );
+	if ( wpOrgShortSource ) {
+		const basename = encodeURIComponent( wpOrgShortSource[ 1 ] );
+		const url =
+			sourceType === 'core'
+				? `https://wordpress.org/${ basename }.zip`
+				: `https://downloads.wordpress.org/${ sourceType }/${ basename }.zip`;
```

Comment:
If you use a `wp.org/` source in `"mappings"` it will construct a URL like `ttps://downloads.wordpress.org//gutenberg.zip` since the `sourceType` is undefined.